client/services/FeedbackGeneratorService.js

```diff
@@ -52,6 +52,21 @@ tie.factory('FeedbackGeneratorService', [
               ].join(''), false);
             }
           }
+          // TODO(eyurko): Make this fancier. Add in more data points.
+          var performanceTests = prompt.getPerformanceTests();
+          for (var i = 0; i < performanceTests.length; i++) {
+            testIndex = i + correctnessTests.length;
+            var expectedPerformance = performanceTests[i].getExpectedPerformance();
+            var observedOutput = observedOutputs[testIndex];
+
+            if (expectedPerformance !== observedOutput) {
+              return FeedbackObjectFactory.create([
+                'Your code is running pretty slowly. Are you sure that it\'s ',
```

Comment:
Well, you haven't really asked them before what they think the performance is, so "Are you sure" is a bit odd.

Maybe: Could you modify it so that it runs in XXX time?